lib/katello/tasks/upgrades/4.1/sync_noarch_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.1' do
+      desc "Synchronize arches field of content in candlepin which should be unset."
+      task :sync_noarch_content => ['environment'] do
+        roots = Katello::RootRepository.where(arch: 'noarch')
```

Comment:
This query needs a bit more refinement to exclude Red Hat content (ie content that has come from your manifest). Updating RH content raises a 403 error with candlepin since it's locked for updating. You probably didn't hit that since you don't have a manifest yet!

This is an example of how you can limit the rootrepositories which are mapped to Custom products - ie non-RH products: `Katello::RootRepository.joins(:product).merge(Katello::Product.custom)`

The error I see for reference:

```
RestClient::Forbidden: Katello::Resources::Candlepin::Content: 403 Forbidden {"displayMessage":"content \"10313\" is locked","requestUuid":"b78e21a1-9883-4047-b405-271288fef649"} (PUT /candlepin/owners/Default_Organization/content/10313)
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/abstract_response.rb:249:in `exception_with_response'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/abstract_response.rb:129:in `return!'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:836:in `process_result'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:743:in `block in transmit'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:727:in `transmit'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:163:in `execute'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:63:in `execute'
/home/vagrant/foreman/.vendor/ruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/resource.rb:79:in `put'
/home/vagrant/katello/app/lib/katello/http_resource.rb:115:in `put'
/home/vagrant/katello/app/lib/katello/resources/candlepin/content.rb:33:in `update'
/home/vagrant/katello/lib/katello/tasks/upgrades/4.1/sync_noarch_content.rake:8:in `block (5 levels) in <top (required)>'
```